packages/nextjs/src/index.server.ts

```diff
@@ -118,6 +118,11 @@ function addServerIntegrations(options: NextjsOptions): void {
   });
   integrations = addOrUpdateIntegration(defaultRewriteFramesIntegration, integrations);
 
+  const nativeBehaviourOnUncaughtException = new Integrations.OnUncaughtException();
+  integrations = addOrUpdateIntegration(nativeBehaviourOnUncaughtException, integrations, {
+    _options: { exitEvenWhenOtherOnUncaughtExceptionHandlersAreRegistered: false },
+  });
```

Comment:
I actually really don't like this. Seems very fragile.

Reply:
It's a little hacky, it's true. It predates my work a few months ago cleaning up how default integrations interact with custom ones, and I thought about changing it then, but I couldn't come up with a better solution to the problem of sometimes needing to merge our default options with user-provided options in the same integration, and sometimes needing to force certain integrations to be there even if the user has overridden the other defaults, without changing the entire system.

If you have a good idea here, I'm all ears. That said, if we have the right tests in place, any changes to the underlying integration class's options should cause the tests to fail, so I don't think it's actually that brittle as long as we're testing well.